hooks/commands/run.sh

```diff
@@ -34,14 +34,18 @@ compose_force_cleanup() {
     plugin_prompt_and_run docker rm -f "$remove_volume_flag" "$(docker_compose_container_name "$adhoc_run_container_name")" || true
   else
     # `compose down` doesn't support force removing images, so we use `rm --force`
-    if [[ "${BUILDKITE_PLUGIN_DOCKER_COMPOSE_LEAVE_VOLUMES:-false}" == "true" ]]; then
-      run_docker_compose rm --force --all -v || true
+    if [[ "${BUILDKITE_PLUGIN_DOCKER_COMPOSE_LEAVE_VOLUMES:-false}" == "false" ]]; then
+      run_docker_compose rm --force -v || true
     else
-      run_docker_compose rm --force --all || true
+      run_docker_compose rm --force || true
```

Comment:
> Which was probably dodgy of me to sneak in.

I was going to ask if we needed another version check in here… but perhaps it's overkill given this is a new plugin?